modules/core/src/main/scala/org/tessellation/infrastructure/trust/TrustStorageUpdater.scala

```diff
@@ -0,0 +1,39 @@
+package org.tessellation.infrastructure.trust
+
+import cats.effect.std.Supervisor
+import cats.effect.{Async, Ref}
+import cats.syntax.flatMap._
+import cats.syntax.foldable._
+import cats.syntax.functor._
+import cats.syntax.traverse._
+
+import scala.concurrent.duration.DurationInt
+
+import org.tessellation.domain.trust.storage.TrustStorage
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.sdk.domain.Daemon
+import org.tessellation.sdk.domain.gossip.Gossip
+
+trait TrustStorageUpdater[F[_]] {
+  def update: F[Unit]
+}
+
+object TrustStorageUpdater {
+
+  def make[F[_]: Async](
+    getOrdinal: Ref[F, Option[SnapshotOrdinal]],
+    gossip: Gossip[F],
+    storage: TrustStorage[F]
+  ): TrustStorageUpdater[F] = new TrustStorageUpdater[F] {
+    override def update: F[Unit] = for {
```

Comment:
`override` modifier is not required as it implements the method, not overrides it.

Reply:
Removed.

Reply:
have you pushed a change?

![image](https://github.com/Constellation-Labs/tessellation/assets/4959931/b4aa37e6-2b3d-4913-a85c-1af4ffbdc8ff)

Reply:
It is now.